projects/ng-core-tester/src/app/app.component.ts

```diff
@@ -99,11 +103,25 @@ export class AppComponent implements OnInit {
       }
     }
 
-    changeLang(item) {
+    changeLang(item: any) {
       this.translateService.use(item.name);
       delete(this.activeLanguagesMenuItem.active);
       item.active = true;
       this.activeLanguagesMenuItem = item;
     }
 
+    private initializeEvents() {
+      this.recordService.onCreate$.subscribe((record: any) => {
+        const pid = record.message.record.pid;
```

Comment:
I changed this, because it did not match the structure of returned object : 

```
export interface RecordEvent {
  resource: string;
  message: object;
}
```

Could you please check the structure of the emitted data ?